cmd/earth/main.go

```diff
@@ -171,7 +171,21 @@ func main() {
 		color.NoColor = true
 	}
 
-	app := newEarthApp(ctx, conslogging.Current(colorMode))
+	padding := conslogging.DefaultPadding
+	customPadding, ok := os.LookupEnv("TARGET_PADDING")
```

Comment:
rather than pull out the env variable here, one can define it as a flag under main. e.g. https://github.com/earthly/earthly/blob/7d72007a596506b3d3925aa2a4bed91e49cc7c4f/cmd/earth/main.go#L320-L326

the flag can also have an env name associated with it.

Reply:
Roger

Reply:
Actually, those look like they get set _after_; which is also why color is doing it the same way?

Reply:
Digging deeper, those could all be handled in an `app.before`

Reply:
ah, I didn't realize the other logging options were set via a regular call to LookupEnv.

Do we even want to support customizing this behaviour? what if We just switched to your new alignment for all cases, then we don't have to document these new options. It might be a bit too much choice?

Reply:
I think we should for CI logs; which is also why the color options exist it looks like: https://docs.earthly.dev/guides/ci-integration#step-4-optional-force-or-disable-color-output

We can leave the 20 char width as the default; and add documentation to the same section?